### ISSUE TITLE
how to implement emoji as an action button

### ISSUE THREAD

**Issue by cool2apps** [prahack/chat_bubbles#9]
Thanks for this wonderful plugin. I would like to add emoji add button to MessageBar like above. But I could not figure out how to implement this. Is it possible to add as an InkWell object? How should I proceed? Thanks.

```
      actions: [
        InkWell(
          child: Icon(
            Icons.emoji,
            color: Colors.black,
            size: 24,
          ),
          onTap: () => openEmojis(),
        ),
      ]
```

**Comment by prahack** [comment]
@cool2apps  Yes. You can use InkWell as an action, the above implementation is correct. As the next step, you need to implement a widget to show emojis inside the `openEmojis()` method. You can use this video to get an idea - https://youtu.be/EzIdiqfWgog
